packages/chatbot-server-mongodb-public/src/processors/formatUserMessageForGeneration.test.ts

```diff
@@ -9,105 +9,115 @@ describe("formatUserMessageForGeneration", () => {
   const userMessage = "Hello, world!";
 
   it("formats front matter correctly for mongodb.com origin", () => {
+    const origin = "https://mongodb.com";
     const result = formatUserMessageForGeneration(userMessage, {
-      origin: "https://mongodb.com",
-    });
+      origin,
+    } as ConversationCustomData);
```

Comment:
```suggestion
    } satisfies ConversationCustomData);
```

prefer `satisfies` as it still uses the typechecker